scripts/tank_cmd.py

```diff
@@ -1104,7 +1104,10 @@ def run_engine_cmd(log, pipeline_config_root, context_items, command, using_cwd,
         console_logout()
         sys.exit()
```

Comment:
The default exit value for sys.exit is 0.

Actually, straight from the Python doc: The optional argument arg can be an integer giving the exit status (defaulting to zero)